account_bank_statement_import_camt/camt.py

```diff
@@ -67,6 +67,7 @@ def parse_transaction_details(self, ns, node, transaction):
         self.add_value_from_node(
             ns, node, [
                 './ns:RmtInf/ns:Ustrd',
+                './ns:RltdPties/ns:CdtrAcct/ns:Tp/ns:Prtry',
```

Comment:
To be on the safe side, can you add this 'account type' field at the bottom of the list so that the actual message fields have a higher priority?